index.bs

```diff
@@ -3245,16 +3245,16 @@ Classes of Changes</h4>
 			For [=Recommendation-track=] [=technical reports=] specifically,
 			this constitutes changes that do not affect conformance,
 			i.e. changes that reasonable implementers
-			would not interpret as changing architectural
+			would not be interpreted as changing architectural
```

Comment:
The subject of this phrase (`reasonable implementers`) is acting on the object (`changes`). `Reasonable implementers would not interpret these changes as changing architectural or interoperability requirements.`
```suggestion
			would not interpret as changing architectural
```